assets/package.json

```diff
@@ -4,8 +4,12 @@
   "version": "0.0.1",
   "license": "UNLICENSED",
   "private": true,
-  "main": "gulp.js",
+  "main": "assets/gulp.js",
   "author": "Miranj",
+  "scripts": {
+    "dev": "cd assets && gulp",
```

Comment:
This assumes the `package.json` file is in the root folder, not in the `assets` folder. But right now on this branch the package file is still in the assets folder, so these commands will fail, right?

Reply:
This needs to be done after moving the `package.json` to _root_ in a different branch. I have removed the modification.